### PR TITLE
2021 02 26 tarkanse planner doc updates

### PR DESCRIPTION
**Pull request by TarkanSevilmis**
Updating documentation for a few fixes and changing the permission scope requirements

### CODE REVIEW

File: changelog/Microsoft.Office.Tasks.json [hunk 1]
```diff
@@ -1,6 +1,58 @@
 {
   "changelog": [
     {
+      "ChangeList": [
+        {
```
Comment (Lauragra) [inline changelog/Microsoft.Office.Tasks.json:5]: Do you need to mention anything about the Prefer header in the request, or is that just a content fix?

Comment (TarkanSevilmis) [reply changelog/Microsoft.Office.Tasks.json:5]: Just content. It is part of protocol, the sample was not including it, so there are comments about samples being wrong.

File: changelog/Microsoft.Office.Tasks.json [hunk 2]
```diff
@@ -1,6 +1,58 @@
...
+          "Description": "Added support for delegated permission of `Tasks.Read` to the read operations of all Planner resources.",
+          "Target": "plannerPlan,plannerTask,plannerPlanDetails,plannerTaskDetails,plannerBucket,plannerAssignedToTaskBoardTaskFormat,plannerBucketTaskBoardTaskFormat,plannerProgressTaskBoardTaskFormat, plannerRoster,plannerRosterMember"
+        },
+        {
+          "Id": "6aa8d44e-1415-43ef-b71f-c680382941ea",
+          "ApiChange": "Permission",
+          "ChangedApiName": "Tasks.ReadWrite",
+          "ChangeType": "Addition",
+          "Description": "Added support for delegated permission of `Tasks.ReadWrite` to the read and write operations of all Planner resources.",
+          "Target": "plannerPlan,plannerTask,plannerPlanDetails,plannerTaskDetails,plannerBucket,plannerAssignedToTaskBoardTaskFormat,plannerBucketTaskBoardTaskFormat,plannerProgressTaskBoardTaskFormat, plannerRoster,plannerRosterMember"
```
Comment (Lauragra) [inline changelog/Microsoft.Office.Tasks.json:19]: We should follow the guidance here for now to keep the Target attribute single-valued, so you'll need to create separate ChangeList entries for each resource that you've added the new permission to.

File: changelog/Microsoft.Office.Tasks.json [hunk 3]
```diff
@@ -1,6 +1,58 @@
...
+          "ChangedApiName": "Tasks.ReadWrite",
+          "ChangeType": "Addition",
+          "Description": "Added support for delegated permission of `Tasks.ReadWrite` to the read and write operations of all Planner resources.",
+          "Target": "plannerPlan,plannerTask,plannerPlanDetails,plannerTaskDetails,plannerBucket,plannerAssignedToTaskBoardTaskFormat,plannerBucketTaskBoardTaskFormat,plannerProgressTaskBoardTaskFormat"
+        }
+      ],
+      "Id": "79526dea-b980-4116-9a6a-81e1ea644efa",
+      "Cloud": "prd",
+      "Version": "v1.0",
+      "CreatedDateTime": "2021-03-16T00:00:00",
```
Comment (Lauragra) [inline changelog/Microsoft.Office.Tasks.json:51]: Just calling out that the date should match the date that we publish the PR. I can update it when this is set to *ready to merge*, unless you want to publish it today, in which case, please update the date when you add the rest of the ChangeList items.

**Pull request merged**